core/src/main/scala/besom/internal/Output.scala

```diff
@@ -77,15 +77,22 @@ trait OutputFactory:
     coll: CC[A]
   )(
     f: A => Output[B]
-  )(using BuildFrom[CC[Output[B]], B, To], Context): Output[To] = sequence(coll.map(f).asInstanceOf[CC[Output[B]]])
+  )(using BuildFrom[CC[Output[B]], B, To], Context): Output[To] = sequence(
+    coll.iterator.map(f).asInstanceOf[CC[Output[B]]]
+  )
   def fail(t: Throwable)(using Context): Output[Nothing] = Output.fail(t)
 trait OutputExtensionsFactory:
   implicit final class OutputSequenceOps[A, CC[X] <: IterableOnce[X], To](coll: CC[Output[A]]):
     def sequence(using BuildFrom[CC[Output[A]], A, To], Context): Output[To] =
       Output.sequence(coll)
   implicit final class OutputTraverseOps[A, CC[X] <: IterableOnce[X]](coll: CC[A]):
     def traverse[B, To](f: A => Output[B])(using BuildFrom[CC[Output[B]], B, To], Context): Output[To] =
-      coll.map(f).asInstanceOf[CC[Output[B]]].sequence
+      coll.iterator.map(f).asInstanceOf[CC[Output[B]]].sequence
+  implicit final class OutputOptionOps[A](output: Output[Option[A]]):
+    def getOrElse[B >: A](default: => B)(using Context): Output[B] =
```

Comment:
IMO it would be nice if a user could also pass values wrapped in `Output` as an argument as well
```suggestion
    def getOrElse[B >: A](default: => B | Output[B])(using Context): Output[B] =
```

Reply:
reimplemented according to the suggestion